packages/generators/src/handlers/default.ts

```diff
@@ -10,21 +10,27 @@ const resolveFile = (file: string): string => {
  * Asks questions to the user used to modify generation
  * @param self Generator values
  * @param Question Contains questions
+ * @param config Configuration for default handler
  */
 
 export async function questions(
   self: CustomGenerator,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Question: Record<string, any>,
+  config: { forceSelection: boolean; skip: Record<string, boolean> } = {
+    forceSelection: false,
+    skip: {},
+  },
```

Comment:
I mean we don't need `forceSelection`, you can pass answers directly and avoid ask question if you pass answers, example:
```
answers: { 
  langType: "ES6",
  devServer: true,
} 
```

and etc

Reply:
Variable just removes 'none' as possible answer, user an still select typescript for langType for example.

Reply:
Yup, thats the point. But in default template it does makes sense. So, therefore need of this variable, so that we can reuse default handler's questions.

Reply:
If something in `answers`, it is already forced, no need to do complex and confusing logic based on different variables

Reply:
Again saying,  answer for a question is not being forced, we are only enforcing that answer can't be `none` (It can be anything other than `none`) and introducing the variable is the simplest possible solution to it, not a complicating one.

Reply:
Why we should ask questions, if we know what we want?

Reply:
We don't know!
That is why we are asking question. We just know it can't be `none`.

Reply:
Why we need skip in this case? I really don't understand your logic

Reply:
Hmm it is simple, please answer this question.

I want to ask user whether they want to use ES6 or Typescript. So why should I skip this question?

Reply:
Let's use better names and structures - we can add `override: { langType: {  variants: [...], default: 'default', force: true } }`, it will allow to override whole questions

Reply:
Okay 👍🏽

Reply:
@alexander-akait Updated it, I guess new change struck nice balance between simplicity and verbosity for configuring it. wdyt?

Reply:
Yes, now we have simple and flexible logic

Reply:
Closing this.